packages/shared/src/features/onboarding/shared/Reviews.tsx

```diff
@@ -26,9 +26,11 @@ export function ReviewCard({
         className,
       )}
     >
-      <h3 className="font-bold typo-callout">{title}</h3>
+      <h3 className="font-bold text-text-primary typo-callout">{title}</h3>
       <p className="text-text-secondary typo-footnote">{content}</p>
-      <span className="font-bold typo-footnote">{author}</span>
+      <span className="font-bold text-text-primary typo-footnote">
+        {author}
+      </span>
```

Comment:
Why not using the Typography element? 🙈

Reply:
Yes def a better alternative

Reply:
Updated ✔️

Reply:
Probably because vibe coding doesn't use the component. Maybe add it to the ruleset?

Reply:
I think it was vibe coded before, for this feature I've not updated it in order to make less line changes 👀